### ISSUE TITLE
New project started

### ISSUE THREAD

**Issue by PedanticHacker** [niklasf/python-chess#238]
I started a new chess GUI project on GitHub named Chess Titan. You can fork it and help me make it the best chess GUI there is. Any help is highly appreciated. Thank you. 👍

**Comment by Mk-Chan** [comment]
Might want to check this thread out if you haven't http://talkchess.com/forum/viewtopic.php?t=65485

**Comment by PedanticHacker** [comment]
Thanks. 👍🏻

**Comment by Mk-Chan** [comment]
Oh one more thing, Chess Titans is the name of another chess software by Microsoft. Maybe be careful with that name (I know yours is Chess Titan but similar enough)

**Comment by PedanticHacker** [comment]
Oops, you're right! I don't wanna get sued. When there was wxWindows (a GUI framework written in C++ now known as wxWidgets), Microsoft threated with a lawsuit if they don't change the name of their framework. So now we have wxWidgets instead of wxWindows. Maybe I'll avoid all this and name my app something else before any Microsoft representative contacts me.

So, got any idea of a cool chess GUI name?

**Comment by Mk-Chan** [comment]
I don't think you should make an issue about something not directly related to python-chess here

**Comment by PedanticHacker** [comment]
Just wanted to grab python-chess devs attention to maybe help me with my project as well. Didn't know how else to do it.

**Comment by sshivaji** [comment]
Hi,

Congrats on starting a new project. I have a whole chess GUI in python,
kivy-chess. I have a lot of experience and feedback on doing a new project
:) Wish you the best of luck. I will follow your project.

**Comment by PedanticHacker** [comment]
Thank you! :)  I have removed the Chess Titan repo because of a possible lawsuit from Microsoft. They have an application named Chess Titans, which is dangerously similar to Chess Titan, and I didn't want to risk it. Now the name of my repo is Chess Fighter. ;)  Would you like to be a collaborator for my Chess Fighter project?

**Comment by sshivaji** [comment]
I think you guys are crazy!

Microsoft will not sue you at this stage. They wont even sure you quickly
if you are a public company! If Microsoft sues you, you will be very lucky!

Can you send me a github link when you are ready?

Best Regards, Shiv

**Comment by PedanticHacker** [comment]
Sure, my friend, here is the link: https://github.com/PedanticHacker/ChessFighter

**Comment by cja000** [comment]
You should have a look to LucasChess (https://github.com/lukasmonk/lucaschess)
I think it is quite similar to what you are trying to do and it is a very good program, already using python-chess :-)

**Comment by PedanticHacker** [comment]
Thanks. I’ll definitely take a look.
